docs/source/nlp/machine_translation.rst

```diff
@@ -508,7 +508,7 @@ Note the ``+`` symbol is needed if we're not adding the arguments to the YAML co
 Megatron
 ^^^^^^^^
 
-We have provided a `Megatron config file <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/machine_translation/conf/megatron.yaml>`__
+We have provided a `Megatron config file <https://github.com/NVIDIA/NeMo/blob/stable/examples/nlp/machine_translation/conf/megatron.yaml>`__
```

Comment:
This link doesn't work. @ericharper was this config added after 1.0 release ?

Reply:
@titu1994 could I merge it? Or we shall wait until the **stable** branch is ready?

Reply:
Yes, this is in 1.1